packages/web/lib/apollo.tsx

```diff
@@ -122,13 +115,9 @@ export function withApollo<PageProps extends object, PageInitialProps = PageProp
           }
 
           if (ssrContext.redirectTarget && ctx?.res) {
-            (ctx.res as any).redirect(ssrContext.redirectTarget)
+            ;(ctx.res as any).redirect(ssrContext.redirectTarget)
             return pageProps
           }
-
-          // getDataFromTree does not call componentWillUnmount
-          // head side effect therefore need to be cleared manually
-          Head.rewind()
```

Comment:
> `Head.rewind has been a no-op since Next.js 9.5, in Next.js 11 it was removed. You can safely remove your usage of Head.rewind.
https://nextjs.org/docs/upgrading